src/main/java/hudson/plugins/jira/JiraBuildAction.java

```diff
@@ -39,23 +42,33 @@ public String getUrlName() {
         return "jira";
     }
 
+    @Exported(inline = true)
+    public Set<JiraIssue> getIssues() {
+        return issues;
+    }
+
+    @Exported
+    public String getServerURL() {
+        JiraSite jiraSite = JiraSite.get(owner.getParent());
+        URL url = Objects.firstNonNull(jiraSite.alternativeUrl, jiraSite.alternativeUrl);
```

Comment:
Oops that is not correct. There's also a possible NPE here with jiraSite being null. Fixed in https://github.com/jenkinsci/jira-plugin/pull/113